src/main/frontend/components/block.cljs

```diff
@@ -2816,13 +2821,27 @@
          (ui/foldable
           [:div.custom-query-title.flex.justify-between.w-full
            [:div.flex.items-center
-            (when (:full-text-search? state)
-              [:a.control.fade-link.mr-1.inline-flex
-               {:title "Refresh query result"
-                :on-mouse-down (fn [e]
-                                 (util/stop e)
-                                 (trigger-custom-query! state))}
-               (ui/icon "refresh" {:style {:font-size 20}})])
+            (when (or (:full-text-search? state)
+                      (and query-time (> query-time 80)))
+              (ui/tippy
```

Comment:
This feature could be super useful for mobile. 
Since tippy is disabled by default on mobile, users won't be able to trigger the query manually?

Reply:
My bad. Only the tooltip is not shown on mobile. Cool.